packaging/linux/build_binaries.sh

```diff
@@ -5,6 +5,7 @@ set -e -u -o pipefail
 here="$(dirname "$BASH_SOURCE")"
 this_repo="$(git -C "$here" rev-parse --show-toplevel)"
 kbfs_repo="$(dirname "$this_repo")/kbfs"
+updater_repo="$(dirname "$this_repo")/go-updater"
```

Comment:
I didn't include this in CI, but now we probably should. That's done in the script that calls this one, `build_and_push_packages.sh`, using your `wait-ci` command.